### DISCUSSION TITLE
微妙网 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#167] [模板讨论]
微妙网 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
用户：【】

用户名没有获取到

**Reply by wjf0214** [reply]
已更新。

**Comment by miaoxiaozhao** [comment]
Failed at 3/4 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.wmiao.com/home.php?mod=spacecp&ac=credit
